<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerAPI test</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>125</width>
     <height>149</height>
    </rect>
   </property>
   <property name="title">
    <string>ConnectState</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbDisconnected">
      <property name="text">
       <string>Disconnected</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbConnecting">
      <property name="text">
       <string>Connecting</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbConnected">
      <property name="text">
       <string>Connected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbDisconnecting">
      <property name="text">
       <string>Disconnecting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cbIsOnline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>77</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>IsOnline</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFetchResources">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Fetch Resources</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMyIp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>MyIP request</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheckUpdate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckUpdate request</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
